Python/Product/VSInterpreters/Interpreter/CondaEnvironmentFactoryProvider.cs

```diff
@@ -483,7 +485,7 @@ private sealed class DiscoverOnDispose : IDisposable {
             public void Dispose() {
                 Interlocked.Decrement(ref _provider._ignoreNotifications);
                 if (_forceDiscovery) {
-                    _provider.ForceDiscoverInterpreterFactories();
+                    _provider.ForceDiscoverInterpreterFactories().DoNotWait();
```

Comment:
seems strange to run discovery when in dispose. maybe they just want to clear the list?

Reply:
Yeah not sure why.